pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -82,7 +82,6 @@ https://github.com/romgrk/barbar.nvim/,,
 https://github.com/utilyre/barbecue.nvim/,,
 https://github.com/chriskempson/base16-vim/,,
 https://github.com/nvchad/base46/,HEAD,
-https://github.com/IogaMaster/neocord.git,main,
```

Comment:
why the removal of those ?

Reply:
No idea. I haven't even noticed them. I just ran `vim-plugins-updated add Canop/nvim-bavon`.
It looks like an alphabetical re-ordering.